api/WebGLVertexArrayObjectOES.json

```diff
@@ -23,7 +23,7 @@
           "opera": "mirror",
           "opera_android": "mirror",
           "safari": {
-            "version_added": false
+            "version_added": "15"
```

Comment:
This interface is actually referring to the vertex array object available through the `OES_vertex_array_object` WebGL 1 extension: https://developer.mozilla.org/en-US/docs/Web/API/OES_vertex_array_object/createVertexArrayOES

We have another definition for the WebGL 2 version: "WebGLVertexArrayObject"

(The interface was basically renamed when the extension was merged into WebGL 2, but we have files for both versions.)

Reply:
Oh yes, good catch. 
I found https://github.com/WebKit/WebKit/commit/ec5c40d6c3660f9c8abaa770cf27adc2f98e6330 for Safari Desktop and https://github.com/WebKit/WebKit/commit/36603fa3a37d279cca03a7165b9efe556a799bc2 for iOS.